internal/pkg/config/types.go

```diff
@@ -19,6 +19,19 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 )
 
+func ListDefaultServices() map[string]string {
```

Comment:
this function seems more like it should be implemented in go-mod-core-contracts, and possibly there it could use go-mod-registry to implement dynamic discovery if there are more clients, because for example how do we add rulesengine to the list of services that the SMA handles and also now security-proxy-setup? Not something to be fixed in this PR, but deserves an issue I think

Reply:
https://github.com/edgexfoundry/edgex-go/issues/1654